db/Catalog/README.md

```diff
@@ -0,0 +1,22 @@
+# catalog
+**catalog** is supposed to mimic a real-life example of a metaphor use-case. It represents a target database schema which stores metadata for tables and columns. The end-users are supposed to provide metadata regarding the systems/tables/columns that they want their target database to hold. An automated process can then read the normalised catalog structures and actually perform the create statements to sync up with the physical database.
+
+## Table dictionary
+
+`catalog.system`: Defines source systems that may contain multiple tables
+`catalog.instance`: Defines instances of source systems
```

Comment:
Can we add in each of these a double spacing or a break-line character ? They don't look like a list when rendering the markdown file

Reply:
Sure - converted that to a list 👍